### ISSUE TITLE
GetLayout doesn't work on gen 1 devices.

### ISSUE THREAD

**Issue by JLOrwell** [xled/xled-docs#33]
As far as i can tell, they never received a code change to provide this info even though their firmware versions are higher  than those listed in this doc as having that info.

**Comment by scrool** [comment]
Are you able to tell from which Gen2 FW version GetLayout works?

**Comment by JLOrwell** [comment]
No. I don't have any devices older than 2.8.3 but I would assume the listed
minimum is correct for gen 2.  I have no reason to doubt it, but both I and
a  user of Artnet2twinkly confirmed gen 1 latest firmware, 2.3.8, does not
have that endpoint.  It returns a 404.  I ended up having to do what scrool
did for the lights themselves, and reverse engineer the web api they claim
doesn't exist.
